fhireval/data/BasicTest.json

```diff
@@ -0,0 +1,328 @@
+{
```

Comment:
Could you re-ident this file with 2 spaces?

Reply:
done